include/GafferUI/AnimationGadget.h

```diff
@@ -74,6 +72,9 @@ class GAFFERUI_API AnimationGadget : public Gadget
 		void setContext( Gaffer::Context *context );
 		Gaffer::Context *getContext() const;
 
+		Gaffer::Set *selectedKeys();
```

Comment:
We should make a note of this new method in the API section of Changes.md.

Reply:
will do.